avocado/core/xunit.py

```diff
@@ -24,6 +25,9 @@
 # We use a subset of the XML format defined in this URL:
 # https://svn.jenkins-ci.org/trunk/hudson/dtkit/dtkit-format/dtkit-junit-model/src/main/resources/com/thalesgroup/dtkit/junit/model/xsd/junit-4.xsd
 
+PRINTABLE = string.ascii_letters + string.digits + string.punctuation + '\n\r '
```

Comment:
Why not to use only `strings.printable`? It's almost the same of all above together, but it also includes all characters that are considered whitespace:

    >> PRINTABLE
    'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!"#$%&\'()*+,-./:;<=>?@[\\]^_`{|}~\n\r '
    >> strings.printable
    '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!"#$%&\'()*+,-./:;<=>?@[\\]^_`{|}~ \t\n\r\x0b\x0c'